sources/getRole.ts

```diff
@@ -188,6 +187,13 @@ function getImplicitRole(element: Element): string | null {
 				return "listbox";
 			}
 			return "combobox";
+		case "th": {
+			const scope = element.getAttribute("scope");
+			if (scope && ["row", "rowgroup"].indexOf(scope) !== -1) {
```

Comment:
```suggestion
			if (scope === 'row' || scope === 'rowgroup') {
```
is simpler and doesn't require and intermediate arrays.